JLJSONMapping/Category/NSError+JLJSONMapping.m

```diff
@@ -8,24 +8,24 @@
 
 #import "NSError+JLJSONMapping.h"
 
-NSString * const kObjectMappingDomain = @"com.mydoghatestechnology.objectmapping";
-NSString * const kObjectMappingDescriptionKey = @"JLObjectMappingDescriptionKey";
-NSString * const kObjectMappingFailureReasonKey = @"JLObjectMappingDetailedFailureReasonKey";
```

Comment:
yeah, my old dog used to eat my electronics so I use that domain now.
http://1.bp.blogspot.com/_mR9iR5KWVg8/RcI1Gy6Zd1I/AAAAAAAAAAM/05nSr9ySP54/s320/All.JPG